docs/plugins/event-list.md

```diff
@@ -0,0 +1,31 @@
+---
+id: EventList
+title: List of Plugin Events
+sidebar_label: List of Plugin Events
+slug: /plugins/EventList
+---
+
+
+# List of Plugin Events
+
+- [Create Server](plugins\plugin-events\create-server.md)
+- [Create Server Authorization](plugins\plugin-events\create-server-auth.md)
+- [Create Server Docker Compose](plugins\plugin-events\create-server-docker-compose.md)
+- [Create Server Dot Env](plugins\plugin-events\create-server-dot-env.md)
+- [Create Entity Service](plugins\plugin-events\create-entity-service.md)
+  - [Create Entity Service Base](plugins\plugin-events\create-entity-service-base.md)
+  - [Create Entity Controller](plugins\plugin-events\create-entity-controller.md)
+  - [Create Entity Controller Base](plugins\plugin-events\create-entity-controller-base.md)
+- [Create Package Json](plugins\plugin-events\create-package-json.md)
+- [Create Message Broker](plugins\plugin-events\create-message-broker.md)
+   - [Create Message Broker Client Options Factory](plugins\plugin-events\create-message-broker-client-options-factory.md)
+   - [Create Message Broker NestJS Module](plugins\plugin-events\create-message-broker-nestjs-module.md)
+   - [Create Message Broker Service](plugins\plugin-events\create-message-broker-service.md)
+   - [Create Message Broker Service Base](plugins\plugin-events\create-message-broker-service-base.md)
+   - [Create Message Broker Topic Enum](plugins\plugin-events\create-message-broker-topics-enum.md)
+- [Create Server Docker Compose](plugins\plugin-events\create-server-docker-compose.md)
```

Comment:
missing create dot env and create docker compose db.
the hierarchy should be as follows
[Create Prisma Schema]
[Create Dot Env]
[Create Docker Compose]
[Create Docker Compose DB]